setup.py

```diff
@@ -104,7 +102,7 @@ def get_long_description():
 
 if __name__ == '__main__':
     setup(name='avocado',
-          version=VERSION,
+          version='39.0',
```

Comment:
If I understand it correctly this and the `avocado.spec` file are the only references to the version, the rest is obtained automatically, right? If so then I welcome this change.